package.json

```diff
@@ -5,22 +5,23 @@
   "author": "Ilias Van Peer",
   "license": "MIT",
   "repository": "zwilias/elm-json",
-  "preferGlobal": true,
```

Comment:
`preferGlobal` has been deprecated for a long time, and is even removed in newer npm versions.